src/assets/_project/_blocks/layout/header/_header-location.scss

```diff
@@ -11,14 +11,15 @@
         font-size: 20px;
         margin-left: 10px;
     }
-    .dropdown-toggle {
+    button.dropdown-toggle {
```

Comment:
Should this not take effect if it's a link instead of a button?

Reply:
We need to make the rule more specified to override the default 2px margin for buttons. The current structure is as bellow. We might need to handle them differently.
<button class="dropdown-toggle".....>

Reply:
You can use the `all-states` mixin to increase priority without changing the rule's effect. That's basically what it was created for.